test/e2e/logs_test.go

```diff
@@ -412,4 +415,30 @@ var _ = Describe("Podman logs", func() {
 		logs.WaitWithDefaultTimeout()
 		Expect(logs).To(Not(Exit(0)))
 	})
+
+	It("podman pod logs with container names", func() {
+		podName := "test-pod"
+		containerName1 := "container-1"
+		containerName2 := "container-2"
+
+		testPod := podmanTest.Podman([]string{"pod", "create", fmt.Sprintf("--name=%s", podName)})
+		testPod.WaitWithDefaultTimeout()
+		Expect(testPod).To(Exit(0))
+
+		log1 := podmanTest.Podman([]string{"run", "--name", containerName1, "-d", "--pod", podName, BB, "/bin/sh", "-c", "echo podman"})
+		log1.WaitWithDefaultTimeout()
+		Expect(log1).To(Exit(0))
+
+		log2 := podmanTest.Podman([]string{"run", "--name", containerName2, "-d", "--pod", podName, BB, "/bin/sh", "-c", "echo podman"})
+		log2.WaitWithDefaultTimeout()
+		Expect(log2).To(Exit(0))
+
+		results := podmanTest.Podman([]string{"pod", "logs", "-names", podName})
+		results.WaitWithDefaultTimeout()
+		Expect(results).To(Not(Exit(0)))
+
+		output := results.OutputToStringArray()
+		Expect(output).To(HaveLen(2))
+		Expect(strings.Contains(output[0], containerName1) && strings.Contains(output[0], containerName2)).To(BeTrue())
```

Comment:
Expect(output).To(ContainSubstring(containerName1))
Expect(output).To(ContainSubstring(containerName2))